content/applications/sales/sales/amazon_connector/features.rst

```diff
@@ -49,14 +49,21 @@ The connector is able to:
    downloading monthly fees reports, handling disputes, or issuing refunds must be managed from
    Amazon Seller Central, as usual.
 
+.. warning::
+   On February 19, 2024, in North American marketplaces, :abbr:`FBA (fulfilled by Amazon)` orders
```

Comment:
Capitalization for the Amazon product, according to the [product overview](https://sell.amazon.com/fulfillment-by-amazon)
```suggestion
   On February 19, 2024, in North American marketplaces, :abbr:`FBA (Fulfilled by Amazon)` orders
```